lib/octokit/client/organizations.rb

```diff
@@ -207,6 +209,68 @@ def organization_public_member?(org, user, options = {})
       end
       alias :org_public_member? :organization_public_member?
 
+      # List pending organization invitations
+      #
+      # Requires authenticated organization member.
+      #
+      # @param org [String, Integer] Organization GitHub login or id.
+      # @return [Array<Sawyer::Resource>] Array of hashes representing invitations.
+      # @see https://developer.github.com/v3/orgs/members/#list-pending-organization-invitations
+      #
+      # @example
+      #   @client.organization_invitations('github')
+      def organization_invitations(org, options = {})
+        options = ensure_api_media_type(:org_memberships, options)
+        get "#{Organization.path org}/invitations", options
+      end
```

Comment:
Since we support aliases for the other organization methods, what do you think about supporting:

```ruby
alias :org_invitations :organization_invitations
```